plugins/labhub.py

```diff
@@ -326,3 +328,50 @@ def eligible(user, iss):
                 yield ('The issue is already assigned to someone. Please '
                        'check if the assignee is still working on the issue, '
                        'if not, you should ask for reassignment.')
+
+    @staticmethod
+    def community_state(pr_count: dict):
+        if (sum(pr_count.values())):
+            if any([pr_count.get('coala-bears', 0) >= 2,
+                    pr_count.get('coala', 0) >= 5,
+                    pr_count.get('coala-utils', 0) >= 1]):
+                return 'on fire'
+            else:
+                return 'alive'
+        else:
+            return 'dead'
+
+    @re_botcmd(pattern=r'pr\s+stats\s+(\d+)(?:hours|hrs)')
+    def pr_stats(self, msg, match):
+        hours = match.group(1)
+        pr_count = dict()
+        start = time.time()
+        for count, (name, repo) in enumerate(self.gh_repos.items(), 1):
+            pr_count[name] = len(list(repo.search_mrs(
+                                         created_after=(datetime.datetime.now()
+                                                        - datetime.timedelta(
+                                                              hours=int(hours)))
+                             )))
+            # handle github rate limiting
+            if (count % 30 == 0):
```

Comment:
Why %30?

Reply:
Because github allows 30 requests per minute for search endpoints